src/organisms/TopBar/Resources/Resources.test.tsx

```diff
@@ -777,7 +774,7 @@ describe('Resources', () => {
         await waitFor(
           () =>
             expect(window.open).toHaveBeenCalledWith(
-              PORTAL_URL_WITHOUT_LOCALHOST,
+              'https://client-amplify-portal-.radix.equinor.com',
```

Comment:
This looks abit weird, with the missing parts of the URL, but the test passes so guess its good? :D

Reply:
Yeah Im not 100% sure why we have a "portal without localhost" url?

Reply:
It is to open the "related" portal environment, but localhost applicationdrawer will open development portal, not localhost

Reply:
It also goes to an acutal URL when in prod, instead of the "radix" url

Reply:
Pushed a fix for this now, I initially thought the portal url variable had been removed, but it was only removed in SAM (still in src/atoms/utils/environment in ACL)